### DISCUSSION TITLE
Is the `podman-clean-transient.service` only for root?

### DISCUSSION THREAD

**Discussion by MartinX3** [containers/podman#22316] [Q&A]
I saw that the `podman-clean-transient.service` only exists in the `system` folder but not in the `user` folder.  
Doesn't it work in the rootless mode or was it just forgotten?  

See PR: https://github.com/containers/podman/pull/16892

`transient_store` is true by default.

**Comment by Luap99** [comment]
likely just forgotten, there is nothing in transient store that shouldn't also work also as rootless AFAIK

**Reply by MartinX3** [reply]
Oh, then an admin should transfer this discussion into an issue I think.
Edit: Ah, I need to create an issue from it.
